src/main/java/frc/robot/commands/ShootingPosCommands/AmpPosition.java

```diff
@@ -0,0 +1,25 @@
+package frc.robot.commands.ShootingPosCommands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.ElevatorSubsystem;
+import frc.robot.subsystems.PivotSubsystem;
+
+public class AmpPosition extends Command{
+    PivotSubsystem pivot;
+    ElevatorSubsystem elevator;
+
+    public AmpPosition(PivotSubsystem pivot, ElevatorSubsystem elevator){
+        this.pivot = pivot;
+        this.elevator = elevator;
+
+    }
+
+    @Override
+    public void execute(){
+        pivot.pivotAmpPosition();
+        elevator.ampPosition();
+    }
```

Comment:
These instructions only need to be sent once, so you probably want to put this in `initialize` instead?

You probably want to also want to add the `isFinished()` method to return `true`

Same comment applies for each of these four commands in this PR

Reply:
oooooh, that's smart. sounds good